bobtemplates/plone_addon/src/+package.namespace+/+package.name+/testing.py.bob

```diff
@@ -25,6 +25,9 @@ class {{{ package.browserlayer }}}(PloneSandboxLayer):
 {{% endif %}}
 
     def setUpZope(self, app, configurationContext):
+        # Load any other ZCML from your package' dependencies manually here.
```

Comment:
Load any other ZCML that is required for your tests...
Z3c.autoinclude is not disabled so that you have to add EVERYTHING manually